common/src/main/proto/TransportMessages.proto

```diff
@@ -209,17 +209,18 @@ message PbRequestSlotsResponse {
 message PbRevive {
   string applicationId = 1;
   int32 shuffleId = 2;
-  int32 mapId = 3;
-  int32 attemptId = 4;
-  int32 partitionId = 5;
-  int32 epoch = 6;
-  PbPartitionLocation oldPartition = 7;
-  int32 status = 8;
+  repeated int32 mapId = 3;
```

Comment:
does this change backward compatible?

Reply:
I believe so because Revive and ChangeLocationResponse is only used in client side.